### DISCUSSION TITLE
Make KC_LOCALE not httponly

### DISCUSSION THREAD

**Discussion by SamTV12345** [keycloak/keycloak#14536] [Ideas]
I'd like to have the error pages localized to the KEYCLOAK_LOCALE variable. Therefore I need to change our own language cookie to the cookie mentioned before that is set by Keycloak. Unfortunately that cookie is httponly which means I can't access it via JavaScript in the ftl pages. Is there a possibility to change that behaviour in Keycloak?

**Comment by stianst** [comment]
This cookie should be http-only as its not intended for use outside the Keycloak server.

There's however already an open bug for the info and error pages not picking up the locale properly.

**Comment by SamTV12345** [comment]
That sounds great. Will this future fix also enable the internationalization of the login page?
